src/maintainer/knowledge_base.rst

```diff
@@ -1048,27 +1048,16 @@ the requirement should be changed from ``run`` to ``run_constrained``. Note that
 PyPy builds
 ===========
 
-To use the PyPy builds you can do the following,
+To use the PyPy 3.6 or 3.7 builds you can do the following,
 
 .. code-block:: bash
 
-   conda install python=3.6.*=*_pypy
-
-or,
-
-.. code-block:: bash
-
-   conda install pypy python=3.6
-
-or,
-
-.. code-block:: bash
-
-   conda install pypy
+   conda create -n pypy36  pypy python=3.6
+   conda create -n pypy37  pypy3.7 python=3.7
```

Comment:
To keep it uniform
```suggestion
   conda create -n pypy37  pypy python=3.7
```

Reply:
whoohoo, that now works. A few days ago it didn't.